### ISSUE TITLE
(Feature Request): pre-no-dislike Likes count

### ISSUE THREAD

**Issue by aleksejrs** [Anarios/return-youtube-dislike#297]
### Extension or Userscript?

Both

### Request or suggest a new feature!

How will the extension show ratio now that the archived dislike count is not growing as fast as the YouTube like count is?

### Can you work on this?

- [ ] Yes
- [X] No

**Comment by Diatonator** [comment]
AFAIK the "No dislike" change was rolled out in waves in different populations. 
There was no "instant dislike switch-off" date for everyone. Some people got they dislikes switched off early, but some after a few weeks.
So I'm not sure if any "dislikes before cut-off" feature will show any accurate data. You could show numbers like:
"dislikes before/after the first dislike disable occured ever"
"dislikes after the latest confirmed device had dislikes switched off"
"dislikes after the Youtube API stopped giving out dislikes"
But will it be useful? Are we able to determine the exact dates for the above? 
Maybe this could be some interesting thing for some who are into hardcore statistics, but likely not for the common folk.

**Comment by Diatonator** [comment]
Oh, and there was some "beta-testing" of switching-off dislikes which happened way earlier for some people

**Comment by Anarios** [comment]
Yes, extended statistics are coming. You'll see actual number of votes by extension user, how estimation was made and any archived data if it exists.

**Comment by Diatonator** [comment]
Oh, I guess the opened issue is about the last one:
>dislikes after the Youtube API stopped giving out dislikes
Then yeah, makes sense

**Comment by Noahs-Git** [comment]
@Anarios @aleksejrs Any updates on this?
